backend/utils/discounts.js

```diff
@@ -182,6 +182,8 @@ const validateDiscountOnOrder = async (
   }, 0)
 
   const shipping = get(cart, 'shipping.amount', 0)
+  const taxRate = parseFloat(get(cart, 'taxRate', 0))
+  const totalTaxes = Math.ceil(taxRate * subTotal)
```

Comment:
Math.ceil will round up to nearest integer. I think we should keep at least 2 decimals?

Reply:
We store everything in cents in DB and localStorage. The UI converts it into dollar before showing